ecs-cli/modules/cli/local/create_app.go

```diff
@@ -18,12 +18,39 @@ package local
 import (
 	"fmt"
 
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
 	"github.com/urfave/cli"
 )
 
+const (
+	// taskDefinitionLabelType represents the type of option used to
+	// transform a task definition to a compose file e.g. remoteFile, localFile.
+	// taskDefinitionLabelValue represents the value of the option
+	// e.g. file path, arn, family.
+	taskDefinitionLabelType  = "ecsLocalTaskDefType"
+	taskDefinitionLabelValue = "ecsLocalTaskDefVal"
+)
+
+const (
+	localTaskDefType  = "localFile"
+	remoteTaskDefType = "remoteFile"
+)
+
+const (
+	ecsLocalDockerComposeFileName = "docker-compose.local.yml"
+)
+
 func Create(c *cli.Context) {
 	// 1. read in task def (from file or arn)
 	// 2. parse task def into go object
 	// 3. write to docker-compose.local.yml file
 	fmt.Println("foo") // placeholder
 }
+
+func validateOptions(c *cli.Context) error {
```

Comment:
Similarly, not sure why this function lives in this file?

Reply:
The flags that are being validated apply to all the sub-commands (`create`, `up`, `down`, `ps`). Should this live in some other file? 🤔 I think we decided to move it in `create_app.go` because it's the first command that users will use for local but I can see how that's not a strong reason.